infobip-spring-data-r2dbc-querydsl/src/main/java/com/infobip/spring/data/r2dbc/QuerydslR2dbcRepositoryFactory.java

```diff
@@ -85,9 +88,15 @@ public QuerydslR2dbcRepositoryFactory(R2dbcEntityOperations operations,
         return RepositoryFragment.implemented(simpleJPAQuerydslFragment);
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
this can be moved above line 97 to minimize the scope (since it's merged I'll do it on master)